### ISSUE TITLE
[file storage]

### ISSUE THREAD

**Issue by alwayslovez** [datalayer/jupyter-ui#97]
### Problem

Hi all, just wondering if it is possible to split the storage and use another server instead of the jupyter server as a storage service?
Thus, it can use the storage server to fetch the notebook and send the code to the jupyter server for execution.

**Comment by echarles** [comment]
Yes, that is possible. You can pull the notebook content from any storage and give it to the as a `nbformat` prop to the 
`Notebook` component.

I have added an example to showcase that https://github.com/datalayer/jupyter-ui/blob/main/packages/react/src/examples/NotebookExternalContent.tsx

**Comment by echarles** [comment]
The example also shows how to persist the user changes in a react local state that you can reuse to persist those changes on demand to your external storage.

**Comment by alwayslovez** [comment]
Thank you very much, I will try this example.

**Comment by chenjpu** [comment]
Specify the nbformat property, and after kernel creation, create a ping.ipynb file in the working directory of the JupyterLab service.How to set not to create the default ping.ipynb file

**Comment by echarles** [comment]
> Specify the nbformat property, and after kernel creation, create a ping.ipynb file in the working directory of the JupyterLab service.How to set not to create the default ping.ipynb file?

@chenjpu jupyter-react `0.8.3 fixes this issue. You can try it. If you define a `nbformat` props on the Notebook component, it should no more query the server to save/read the default ping.ipynb.
